azerothcore/azerothcore-wotlk
# fix (core): server.log dump

When Logger.entities.player.dump=4,Console Server is turned on you get alot of weird stuff happening when you delete a character.
![image](https://user-images.githubusercontent.com/16887899/154808707-423c6aef-cae3-41c0-8846-daa855c9cee9.png)
looking at the server.log you will then understand that error code.
![image](https://user-images.githubusercontent.com/16887899/154808732-b1bdbcec-767b-4861-af4b-ecb66800993e.png)
This gets rid of that insanity. It didnt even back up all the things that pdumps does so this one is just a really odd out of date feature/functon.

## Changes Proposed:
-  prevents the server.log from doing some weird sql query in server.log dealing with some alternative pdump when Logger.entities.player.dump=4,Console Server is turned on. 

- This does not effect the pdump command.

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- builds
- performs


## How to Test the Changes:


1. have logger Logger.entities.player.dump=4,Console Server
2. delete character
3. notice sql query is not filling up server.log file.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
